package.json

```diff
@@ -1,45 +1,98 @@
 {
-    "name": "pixi-shadows",
-    "version": "0.0.4",
-    "description": "A plugin for PixiJS V4 that allows you to add dymanic shadows to your stage",
-    "main": "build/shadows/node/pixi-shadows.js",
-    "scripts": {
-        "start": "npm run start:pixi-lights",
-        "start:pixi-lights": "webpack-dev-server --open --config src/demos/pixi-lights/webpack.config.js",
-        "start:basic": "webpack-dev-server --open --config src/demos/basic/webpack.config.js",
-        "start:advanced": "webpack-dev-server --open --config src/demos/advanced/webpack.config.js",
-        "start:system": "webpack-dev-server --open --config src/demos/system/webpack.config.js",
-        "build": "webpack",
-        "build:demos": "npm run build-demo:pixi-lights && npm run build-demo:basic && npm run build-demo:advanced && npm run build-demo:system",
-        "build-demo:pixi-lights": "webpack --config src/demos/pixi-lights/webpack.config.js",
-        "build-demo:basic": "webpack --config src/demos/basic/webpack.config.js",
-        "build-demo:advanced": "webpack --config src/demos/advanced/webpack.config.js",
-        "build-demo:system": "webpack --config src/demos/system/webpack.config.js"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/TarVK/pixi-shadows.git"
-    },
-    "author": "Tar van Krieken",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/TarVK/pixi-shadows/issues"
-    },
-    "homepage": "https://github.com/TarVK/pixi-shadows",
-    "dependencies": {
-        "dat.gui": "^0.7.2",
-        "pixi-layers": "^0.1.9",
-        "pixi-lights": "^2.0.1",
-        "pixi.js": "^4.8.1",
-        "stats-js": "^1.0.0-alpha1"
-    },
-    "devDependencies": {
-        "babel-cli": "^6.26.0",
-        "babel-loader": "^7.1.5",
-        "babel-preset-env": "^1.7.0",
-        "webpack": "^4.16.2",
-        "webpack-cli": "^3.1.0",
-        "webpack-dev-server": "^3.1.5",
-        "webpack-serve": "^2.0.2"
-    }
+  "name": "pixi-shadows",
+  "version": "0.0.5",
```

Comment:
We should bump this to a new major version I feel like, since it's for a completely different version of pixi-js. I have not tested this, but I'm quite sure this will be a breaking change (when using pixi v4).

Reply:
yes. I think after the PR is merged, you can use `npm run release:major`